en/flight_modes_rover/index.md

```diff
@@ -1,16 +1,101 @@
-- [Differential Steering Rover](../frames_rover/differential_rover_v1.md#drive-modes)
+This section outlines all supported drive modes for rovers. Note that certain flight modes have different implementations for the specific modules.
 
-## Further Information
+::: warning
+Selecting any other mode than those below will either stop the rover or can lead to undefined behavior.
+:::
 
-- [Basic Configuration > Flight Modes](../config/flight_mode.md) - How to map RC control switches to specific flight modes
-- [Flight Modes (Multicopter)](../flight_modes_mc/index.md)
-- [Flight Modes (Fixed-wing)](../flight_modes_fw/index.md)
-- [Flight Modes (VTOL)](../flight_modes_vtol/index.md)
+## Manual Mode
+
+The _Manual_ mode stops the rover when the RC control sticks are centred.
+To manually move/drive the vehicle you move the sticks outside of the centre.
+
+[**Differential-steering Rover**](../frames_rover/differential_rover.md): When under manual control the throttle and roll sticks control the _speed_ and _yaw rate_ of the vehicle.
+
+[**Ackermann Rover**](../frames_rover/ackermann_rover.md): When under manual control the throttle and roll sticks control the _speed_ and _steering angle_ of the vehicle.
+
+![Rover Manual Sticks](../../assets/airframes/rover/flight_modes/manual_inputs.png)
+
+Note that the rover does not attempt to maintain a specific orientation or compensate for external factors like slopes or uneven terrain!
+The user is responsible for making the necessary adjustments to the stick inputs to keep the rover on the desired course.
+
+## Acro Mode
+
+::: info
+Acro mode is only supported for differential-steering rovers.
+:::
+
+Acro Mode is similar to [Manual mode](#manual-mode), but with closed-loop yaw rate control.
+In this mode, the left stick input remains open-loop for forward speed control, while the right stick input commands a desired yaw rate setpoint, which is then maintained by the rover's closed-loop control system.
+
+- Left Stick:
+
+  Behavior remains the same as in manual mode, directly controlling the rover's forward speed in an open-loop manner.
+
+- Right Stick:
+
+  - Centered: Rover stops rotating and tries to maintains its current heading.
+  - Pushed left/right: Rover rotates counter-clockwise/clockwise at the rate commanded by the stick input, using a closed-loop controller (e.g., PID) to try to ensure the vehicle yaw rate matches the given setpoint.
+
+See [Tuning(basic)](../frames_rover/differential_rover.md#tuning-basic) to go through the necessary setup to use acro mode for differential-steering rovers.
+
+## Mission Mode
+
+_Mission mode_ is an automatic mode that causes the vehicle to execute a predefined autonomous [mission](../flying/missions.md) plan that has been uploaded to the flight controller.
+The mission is typically created and uploaded with a Ground Control Station (GCS) application, such as [QGroundControl](https://docs.qgroundcontrol.com/master/en/).
+
+### Mission commands

```

Comment:
Add another section `### Mission Parameters` linking to the sections for each vehicle. Unless they are largely shared/common (doesn't appear so) in which case you might  go the other way - have the mission settings here.